app/assets/javascripts/application/glossary.js

```diff
@@ -14,5 +14,8 @@ $(document).on('mouseout', '.glossary-term, .glossary-definition', function(e) {
 });
 
 $(document).on('click', '.glossary-term', function(e) {
-  window.open($(e.target).find('a').attr('href'));
+  if (!$(e.target).closest('.glossary-definition').length) {
```

Comment:
If i understand correctly, the problem is that sometimes the user is clicking a child of `.glossary-term` that does not have an `a`, right?  Like, a sibling of the `a`?  
In that case, would it make sense to say `if (!$(e.target).find('a').length) {`?  That will also cover a case where `.glossary-definition` has no `a`, for instance if it is a very short definition.